src/remote_connector_dao/get_signal.py

```diff
@@ -0,0 +1,44 @@
+import time
+from remote_connector_dao.virtual_channels_builder import (
+    build_coincidences_virtual_channel,
+)
+
+
+def get_single_counts_per_second(timetagger_proxy, tagger_controller, channels_list):
+    tagger_controller.setTriggerLevel(1, 0.09)
+    tagger_controller.setTriggerLevel(2, 0.09)
+
+    with timetagger_proxy.Countrate(tagger_controller, channels_list) as cr:
+        cr.startFor(int(1e12), clear=True)
```

Comment:
sugestão: criar constants para representar esse numero. 
Ajuda na reutilização (vc esta utilizando o mesmo valor na outra função) e evita os chamados ["valores mágicos"](https://www.inf.unibz.it/~calvanese/teaching/04-05-ip/lecture-notes/uni04/node15.html)

Reply:
Valeu pela dica. Vou colocar como variável global no arquivo, mas no futuro queria centralizar essas variáveis globais em um único arquivo. Acho que vai ajudar muito.